sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -414,6 +421,15 @@ DataSource buildDatasource() throws Exception {
       return toBuilder().setCoder(coder).build();
     }
 
+    /**
+     * This method used to set the size of the data that is going to be fetched from the database.
```

Comment:
This description is a bit confusing - it's not the size of the whole data, it's how much will be fetched per every database call and kept in memory. Maybe just link to Statement.setFetchSize.